service/src/main/java/aero/minova/cas/SecurityConfig.java

```diff
@@ -64,28 +88,64 @@ protected void configure(HttpSecurity http) throws Exception {
 		http.httpBasic();
 		http.csrf().disable(); // TODO Entferne dies. Vereinfacht zur Zeit die Loginseite.
 		http.logout().permitAll();
+
 	}
 
-	@Bean
-	public SpringSecurityDialect springSecurityDialect() {
-		return new SpringSecurityDialect();
+	@Order(1)
+	@Configuration
+	@Profile("development")
```

Comment:
Der Profilname könnte kürzer sein: `@Profile("dev")`

Reply:
Achso, weil wir haben uns an die Benamung in der application.properties gehalten.
Dort steht:
spring.profiles.default=development

Sollen wir hier auch "dev" anstelle von "development" nehmen? Wenn ja, müssen wir dann noch an anderen Stellen "dev" aus "development" machen?